components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2173,9 +2169,9 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
             const costCenter = await this.costCenterDB.findById(AttributionId.render(limit.attributionId));
             if (costCenter) {
                 if (limit.reached) {
-                    result.unshift("The spending limit is reached.");
+                    result.unshift("The usage limit is reached.");
                 } else if (limit.almostReached) {
-                    result.unshift("The spending limit is almost reached.");
+                    result.unshift("The usage limit is almost reached.");
```

Comment:
Q: What is the threshold for almost reached?

Reply:
It's 80%.